# Changelog

## [0.1.6](https://github.com/googleapis/google-cloud-python/compare/google-cloud-config-v0.1.5...google-cloud-config-v0.1.6) (2024-02-15)


### Bug Fixes

* [Many APIs] fix `ValueError` in `test__validate_universe_domain` ([#12282](https://github.com/googleapis/google-cloud-python/issues/12282)) ([b985096](https://github.com/googleapis/google-cloud-python/commit/b985096d43add8214172ff993e00293e6c8757cb))
* **deps:** [Many APIs] Require `google-api-core&gt;=1.34.1` ([#12305](https://github.com/googleapis/google-cloud-python/issues/12305)) ([2aa7f17](https://github.com/googleapis/google-cloud-python/commit/2aa7f17a5fd4f2249260225db91fb0414d06eaa7))

## [0.1.5](https://github.com/googleapis/google-cloud-python/compare/google-cloud-config-v0.1.4...google-cloud-config-v0.1.5) (2024-02-06)


### Bug Fixes

* Add google-auth as a direct dependency ([f3db074](https://github.com/googleapis/google-cloud-python/commit/f3db074e7bbf505d5989e4c353461ab6bef4905c))
* Add staticmethod decorator to _get_client_cert_source and _get_api_endpoint ([f3db074](https://github.com/googleapis/google-cloud-python/commit/f3db074e7bbf505d5989e4c353461ab6bef4905c))
* Resolve AttributeError 'Credentials' object has no attribute 'universe_domain' ([f3db074](https://github.com/googleapis/google-cloud-python/commit/f3db074e7bbf505d5989e4c353461ab6bef4905c))

## [0.1.4](https://github.com/googleapis/google-cloud-python/compare/google-cloud-config-v0.1.3...google-cloud-config-v0.1.4) (2024-02-01)


### Features

* Allow users to explicitly configure universe domain ([#12239](https://github.com/googleapis/google-cloud-python/issues/12239)) ([8004d15](https://github.com/googleapis/google-cloud-python/commit/8004d15d9e6baa4dc5bc3f09d528e176d54d9ec5))

## [0.1.3](https://github.com/googleapis/google-cloud-python/compare/google-cloud-config-v0.1.2...google-cloud-config-v0.1.3) (2024-01-19)


### Features

* [google-cloud-config] added Terraform Plan ([#12197](https://github.com/googleapis/google-cloud-python/issues/12197)) ([2de325b](https://github.com/googleapis/google-cloud-python/commit/2de325ba6aef85c98c9ebbe03fc6a4ebb2834a12))

## [0.1.2](https://github.com/googleapis/google-cloud-python/compare/google-cloud-config-v0.1.1...google-cloud-config-v0.1.2) (2023-12-07)


### Features

* Add support for python 3.12 ([b96013d](https://github.com/googleapis/google-cloud-python/commit/b96013d2c31e3602bb885bf8d7296cc49c3a4642))
* Introduce compatibility with native namespace packages ([b96013d](https://github.com/googleapis/google-cloud-python/commit/b96013d2c31e3602bb885bf8d7296cc49c3a4642))


### Bug Fixes

* Require proto-plus &gt;= 1.22.3 ([b96013d](https://github.com/googleapis/google-cloud-python/commit/b96013d2c31e3602bb885bf8d7296cc49c3a4642))
* Use `retry_async` instead of `retry` in async client ([b96013d](https://github.com/googleapis/google-cloud-python/commit/b96013d2c31e3602bb885bf8d7296cc49c3a4642))

## [0.1.1](https://github.com/googleapis/google-cloud-python/compare/google-cloud-config-v0.1.0...google-cloud-config-v0.1.1) (2023-09-19)


### Documentation

* Minor formatting ([9487380](https://github.com/googleapis/google-cloud-python/commit/94873808ece8059b07644a0a49dedf8e2906900a))

## 0.1.0 (2023-08-31)


### Features

* add initial files for google.cloud.config.v1 ([#11608](https://github.com/googleapis/google-cloud-python/issues/11608)) ([c53680f](https://github.com/googleapis/google-cloud-python/commit/c53680f647738b8fc3f9cce86455dd3f195e4ff6))

## Changelog
